src/modules/70-pipeline/pages/execution/ExecutionPolicyEvaluationsView/utils.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { defaultTo, forEach, includes, map } from 'lodash-es'
+import { GovernanceMetadata } from 'services/cd-ng'
+import { Evaluation } from 'services/pm'
+import { EvaluationStatus, IacmFormattedData } from './types'
+
+/* istanbul ignore next */
+const defaultEmpty = (value: string | number | undefined): string => defaultTo(value, '') as string
+
+/* istanbul ignore next */
+const formatIacmGovernanceData = (
+  data?: Evaluation[]
+): {
+  iacmData: IacmFormattedData[]
+  iacmStatus: EvaluationStatus
+} => {
+  const iacmData: IacmFormattedData[] = []
+  let status: EvaluationStatus = EvaluationStatus.PASS
+  const statuses: EvaluationStatus[] = []
+  forEach(data, ({ id, details }) =>
+    forEach(details, policySet => {
+      statuses.push(defaultTo(policySet.status as EvaluationStatus, EvaluationStatus.PASS))
+      iacmData.push({
+        policySetId: defaultEmpty(policySet.identifier),
+        policySetName: defaultEmpty(policySet.name),
+        status: defaultEmpty(policySet.status),
+        identifier: defaultEmpty(policySet.identifier),
+        created: defaultEmpty(policySet.created),
+        accountId: defaultEmpty(policySet.account_id),
+        orgId: defaultEmpty(policySet.org_id),
+        projectId: defaultEmpty(policySet.project_id),
+        policyMetadata: map(policySet.details, policy => ({
+          id: id,
```

Comment:
Not worth fixing, but...
```suggestion
          id,
```